lib/resque/failure/blacklist_exceptions_registrar.rb

```diff
@@ -0,0 +1,32 @@
+module Resque
+  module Failure
+    # A module to access blacklisted exceptions
+    # on a class level
+    module BlacklistExceptionsRegistrar
+      def self.included(base)
+        base.extend ClassMethods
+      end
+
+      # Class methods for accessing
+      # blacklisted exceptions
+      module ClassMethods
+        # Set a list of blacklisted exceptions
+        #
+        # The method sets a list of exceptions which will
+        # not be pushed to the failed queue when a job fails and it's
+        # exception is included to the blacklisted ones.
+        #
+        # @param [Array<String>] exceptions the list of exceptions to blacklist
+        def set_blacklisted_exceptions(exceptions)
+          @blacklist = exceptions
```

Comment:
We should ensure that the given argument is an Array, otherwise the subsequent calls to `include?` will raise an exception.

Reply:
Right, so do you think raising an exception here in case the argument is not a list, is the way to go? This would for sure make the user aware that the `exceptions` needs to be an array. I would also need to ensure that the elements in the array are strings.

Reply:
I'd agree to raise an exception to the user in order to inform him about the allowed argument type.